Add `#width` and `#height` methods to the RMagick processor

mehlah opened the pull request:
This keeps the symmetry with MiniMagick processor where `#width` and
`#height` methods has been introduced by #1405.

mehlah commented on lib/carrierwave/processing/rmagick.rb:
```diff
@@ -351,5 +373,9 @@ def destroy_image(image)
     end
 
+    def rmagick_image
+      ::Magick::Image.ping(current_path).first
```
I used `ping` instead of `read` as we are interested in the image metadata, but don't care about the pixel data.
Ref http://www.rubydoc.info/gems/rmagick/2.15.4/Magick/Image#ping-class_method

thomasfedb replied:
I'm a little concerned about this. Calling `ping` will return an image with invalid pixel data, and naming this `rmagick_image` seems to be asking for trouble.

Either we need to read the entire image, or we should rename the method.

thomasfedb commented:
I'm happy with this change in concept, but I've noted one concern above.

mehlah commented:
@thomasfedb I agree :+1:. I fixed it by using `read` instead of `ping` to keep it consistent with MiniMagick code.
I also added an additional commit to remove an unnecessary `private` keyword introduced after recent merges (see the commit message for details).
Thanks!

thomasfedb commented:
Thanks @mehlah!
